### PR TITLE
update-documentation

### PR DESCRIPTION
**Pull request by priyanka9634**
update-documentation

### CODE REVIEW

File: index.ftd [hunk 1]
```diff
@@ -40,108 +41,192 @@ $processor$: pr.toc
...
-site-name: $site-name
-logo-height: 38
-products: $ps.example-product
+Below are details of all properties of this component.
 
--- end: cb.rendered.output
+-- site-doc.component-property: `site-logo`
 
--- end: cb.rendered
+This is a default optional property of the
```
Comment (gsalunke) [inline index.ftd:114]:
```suggestion

```

**Review by gsalunke — CHANGES_REQUESTED** [review]

File: index.ftd [hunk 2]
```diff
@@ -40,108 +41,229 @@ $processor$: pr.toc
...
+-- site-doc.component-property: `current-page`
+
+Using `sitemap processor`we can 
+added current-page to `header` component.
+
+-- site-doc.component-property: `products`
+
+Added dependency `fifthtry.github.io/product-switcher` 
+in FASTN.ftd. We can added products to 
+`header` component.
```
Comment (gsalunke) [inline index.ftd:172]:
```suggestion
Its used to pass `products` record list to `header` component. `products` property will be passed to `product-switcher` which is imported by default inside `header` component `FASTN.ftd`.
```

File: index.ftd [hunk 3]
```diff
@@ -40,108 +41,229 @@ $processor$: pr.toc
...
+It’s used to add a link to click
+to site logo or site-name.
+
+-- site-doc.component-property: `site-name`
+
+It’s used to add a site name to `header` component.
+
+-- site-doc.component-property: `sections`
+
+Using `sitemap processor` or
```
Comment (gsalunke) [inline index.ftd:137]:
```suggestion
It's used to show `fastn.sitemap` sections.
```

File: index.ftd [hunk 4]
```diff
@@ -40,108 +41,229 @@ $processor$: pr.toc
...
+
+-- site-doc.component-property: `site-name`
+
+It’s used to add a site name to `header` component.
+
+-- site-doc.component-property: `sections`
+
+Using `sitemap processor` or 
+`toc processor` we can added sections to 
+`header` component.
```
Comment (gsalunke) [inline index.ftd:139]:
```suggestion
```

File: index.ftd [hunk 5]
```diff
@@ -40,108 +41,229 @@ $processor$: pr.toc
...
+
+-- site-doc.component-property: `sub-sections`
+
+Using `sitemap processor` or 
+`toc processor` we can added sub-sections
+to `header` component.
```
Comment (gsalunke) [inline index.ftd:145]:
```suggestion
It's used to show `fastn.sitemap` sub-sections.
```

File: index.ftd [hunk 6]
```diff
@@ -40,108 +41,229 @@ $processor$: pr.toc
...
+
+-- site-doc.component-property: `toc`
+
+Using `sitemap processor` or 
+`toc processor` we can added toc to 
+`header` component.
```
Comment (gsalunke) [inline index.ftd:151]:
```suggestion
It's used to show `fastn.sitemap` toc.
```

File: index.ftd [hunk 7]
```diff
@@ -40,108 +41,229 @@ $processor$: pr.toc
...
+
+-- site-doc.component-property: `current-section`
+
+Using `sitemap processor`we can 
+added current-section to `header` component.
```
Comment (gsalunke) [inline index.ftd:156]:
```suggestion
It's used to show `fastn.sitemap` current active section.
```

File: index.ftd [hunk 8]
```diff
@@ -40,108 +41,229 @@ $processor$: pr.toc
...
+
+-- site-doc.component-property: `current-subsection`
+
+Using `sitemap processor`we can 
+added current-subsection to `header` component.
```
Comment (gsalunke) [inline index.ftd:161]:
```suggestion
It's used to show `fastn.sitemap` current active sub-section.
```

File: index.ftd [hunk 9]
```diff
@@ -40,108 +41,229 @@ $processor$: pr.toc
...
+
+-- site-doc.component-property: `current-page`
+
+Using `sitemap processor`we can 
+added current-page to `header` component.
```
Comment (gsalunke) [inline index.ftd:166]:
```suggestion
It's used to show `fastn.sitemap` current active page.
```

File: index.ftd [hunk 10]
```diff
@@ -40,108 +41,229 @@ $processor$: pr.toc
...
+
+-- site-doc.component-property: `products`
+
+Added dependency `fifthtry.github.io/product-switcher` 
+in FASTN.ftd. We can added products to 
+`header` component.
+
+-- site-doc.component-property: `login-button`
+
+It’s used to add a login button to `header` component.
```
Comment (gsalunke) [inline index.ftd:176]:
```suggestion
It’s used to enable login button into `header` component. Login button appears at the right side of header component.
```

File: index.ftd [hunk 11]
```diff
@@ -40,108 +41,229 @@ $processor$: pr.toc
...
+
+-- site-doc.component-property: `pricing-cta`
+
+It’s used to add a pricing button to `header` component.
```
Comment (gsalunke) [inline index.ftd:180]:
```suggestion
```

File: index.ftd [hunk 12]
```diff
@@ -40,108 +41,229 @@ $processor$: pr.toc
...
+
+Added dependency `fifthtry.github.io/product-switcher` 
+in FASTN.ftd. We can added products to 
+`header` component.
+
+-- site-doc.component-property: `login-button`
+
+It’s used to add a login button to `header` component.
+
+-- site-doc.component-property: `pricing-cta`
```
Comment (gsalunke) [inline index.ftd:178]: @priyanka9634 remove this property from `header`

File: index.ftd [hunk 13]
```diff
@@ -40,108 +41,229 @@ $processor$: pr.toc
...
+
+It’s used to add a login button text to `header` component.
+
+-- site-doc.component-property: `sign-in-txt`
+
+It’s used to add a sign-in button text to `header` component.
+
+-- site-doc.component-property: `logo-height`
+
+It’s used to add a height to logo to `header` component.
```
Comment (gsalunke) [inline index.ftd:192]:
```suggestion
It’s used to add a height to logo to `header` component. Its CSS unit is `px`
```

**Review by gsalunke — CHANGES_REQUESTED** [review]

File: index.ftd [hunk 14]
```diff
@@ -42,106 +43,222 @@ $processor$: pr.toc
...
+sections: $sitemap.sections
+sub-sections: $sitemap.subsections
+current-section: $sitemap.current-section
+current-subsection: $sitemap.current-subsection
+current-page: $sitemap.current-page
+toc: $sitemap.toc
+site-logo: $site-logo
+site-url: $site-url
+site-name: $site-name
+logo-height: 38
```
Comment (gsalunke) [inline index.ftd:73]:
```suggestion
logo-height: 38
logo-width: 120
```

File: index.ftd [hunk 15]
```diff
@@ -42,106 +43,222 @@ $processor$: pr.toc
...
+products: $ps.example-product 
+
+-- end: site-doc.output
+
+-- site-doc.component-doc: `header`
+package-link: https://fifthtry.github.io/header
+github-link: https://fifthtry.github.io/header
```
Comment (gsalunke) [inline index.ftd:80]:
```suggestion
github-link: https://github.com/FifthTry/header
```

File: index.ftd [hunk 16]
```diff
@@ -42,106 +43,222 @@ $processor$: pr.toc
...
+
+-- site-doc.component-property: `sign-in-txt`
+
+It’s used to add a sign-in button text to 
+`header` component.
+
+-- site-doc.component-property: `logo-height`
+
+It’s used to add a height to logo to 
+`header` component. Its CSS unit is `px`
```
Comment (gsalunke) [inline index.ftd:191]:
```suggestion
-- site-doc.component-property: `logo-height`

It’s used to add a height to logo to 
`header` component. Its CSS unit is `px`

-- site-doc.component-property: `logo-width`

It’s used to add a width to logo to 
`header` component. Its CSS unit is `px` and default value is `120`

```

File: index.ftd [hunk 17]
```diff
@@ -42,106 +43,222 @@ $processor$: pr.toc
...
+type: String
+default: Default
+
+-- site-doc.table-of-property: `login-btn-txt`
+type: String
+default: Default
+
+-- site-doc.table-of-property: `logo-height`
+type: Integer
+default: Default
```
Comment (gsalunke) [inline index.ftd:255]:
```suggestion
-- site-doc.table-of-property: `logo-height`
type: Integer
default: Optional

-- site-doc.table-of-property: `logo-width`
type: Integer
default: Optional
```

**Review by gsalunke — APPROVED** [review]

File: docs/header-with-login.ftd [hunk 18]
```diff
@@ -0,0 +1,149 @@
+-- import: fastn-community.github.io/site-doc
+-- import: fastn-community.github.io/site-doc
```
Comment (Heulitig) [inline docs/header-with-login.ftd:2]: Remove this import. This is already done once

```suggestion
```

File: docs/header-with-login.ftd [hunk 19]
```diff
@@ -0,0 +1,148 @@
...
+
+-- end: site-doc.output
+
+-- site-doc.component-doc: `header`
+package-link: https://fifthtry.github.io/header
+github-link: https://github.com/FifthTry/header
+package-name: fifthtry.github.io/header
+info: `-- header` can be used as header on your fastn web projects.
+
+The `Header` package(fifthtry.github.io/header). Use inside `FASTN` files
```
Comment (Heulitig) [inline docs/header-with-login.ftd:40]:
```suggestion
The `header` package(fifthtry.github.io/header). Use inside `FASTN` files
```

File: docs/header-with-login.ftd [hunk 20]
```diff
@@ -0,0 +1,148 @@
...
+
+Below are details of all properties of this component.
+
+-- site-doc.component-property: `sections`
+
+It's used to show `fastn.sitemap` sections.
+
+-- site-doc.component-property: `site-url`
+
+It’s used to add a link to click
```
Comment (Heulitig) [inline docs/header-with-login.ftd:83]:
```suggestion
It’s used to add a link
```

File: docs/header-with-login.ftd [hunk 21]
```diff
@@ -0,0 +1,148 @@
...
+property will be passed to `product-switcher` which is imported by default
+inside `header` component `FASTN.ftd`.
+
+-- site-doc.component-property: `login-button`
+
+It’s used to enable login button in `header` component.
+
+-- site-doc.component-property: `login-btn-txt`
+
+It’s used to add login button text in `header` component.
```
Comment (Heulitig) [inline docs/header-with-login.ftd:102]:
```suggestion
It’s used to specify login button text in `header` component.
```

File: docs/header-with-login.ftd [hunk 22]
```diff
@@ -0,0 +1,148 @@
...
+
+-- site-doc.component-property: `login-btn-url`
+
+It’s used to add login button url in `header` component.
```
Comment (Heulitig) [inline docs/header-with-login.ftd:106]:
```suggestion
It’s used to specify login button url in `header` component.
```

File: docs/header-with-login.ftd [hunk 23]
```diff
@@ -0,0 +1,148 @@
...
+
+-- site-doc.header: Properties at a glance:
+
+Below table shows all properties at a glance
```
Comment (Heulitig) [inline docs/header-with-login.ftd:110]:
```suggestion
Below table shows all properties
```

File: docs/header-with-logo-site-name.ftd [hunk 24]
```diff
@@ -0,0 +1,116 @@
+-- import: fastn-community.github.io/site-doc
+-- import: fastn-community.github.io/site-doc
```
Comment (Heulitig) [inline docs/header-with-logo-site-name.ftd:2]: Remove this import

```suggestion
```

**Review by Heulitig — COMMENTED** [review]
Similar content is repeated in several files, so similar corrections are required in all of them

**Pull request merged**